udfs/remote_udfs/nlp/call_nlp/main.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from google.cloud import language_v1
+
+def remote_vertex_ai(request):
+    '''
+    This function serves the request.
+    The additional parameter "mode" is part of a key value pair sent as part of the call.
+    For more information on the UserDefinedContext see:
+    https://cloud.google.com/bigquery/docs/reference/standard-sql/remote-functions#input_format
+    '''
+    request_json = request.get_json()
+    mode = request_json['userDefinedContext']['mode']
+    # The input into the function from BigQuery is retrieved with the calls field. 
+    calls = request_json['calls']
+    if mode == "call_nlp":
+        return call_nlp(calls)
+
+    return json.dumps({"Error in Request": request_json}), 400
```

Comment:
How about

return json.dumps({"errorMessage": f"Not supported 'mode' in user_defined_context: {mode}"}), 400

Reply:
Updated